app/Http/Controllers/Service/Admin/CentreUsersController.php

```diff
@@ -251,15 +252,18 @@ public function download(): void
     }
 
     /**
-     * Handle deleting a centre user
+     * Handle deleting a centre user - change email and name to allow them to be re-added
+	 * later if needed
      * @param int $id
      * @return RedirectResponse
      */
     public function delete(int $id): RedirectResponse
     {
         $centreUser = CentreUser::findOrFail($id);
-        $centreUser->centre->centreUsers()->detach($id);
-        $centreUser->delete();
+		$centreUser->name = 'DELETED' . $centreUser->name;
+		$centreUser->email = 'DELETED' . $centreUser->email;
+		$centreUser->deleted_at = Carbon::now()->format('Ymd-His');
+        $centreUser->save();
```

Comment:
Indenting? tabs vs spaces?

```suggestion
        $centreUser->name = 'DELETED' . $centreUser->name;
        $centreUser->email = 'DELETED' . $centreUser->email;
        $centreUser->deleted_at = Carbon::now()->format('Ymd-His');
        $centreUser->save();
```

Reply:
Urgh I had a nightmare with my settings in PhpStorm not so long ago, so there may be more like that. I'll commit your suggested changes and sort my IDE to prevent this in future. Thanks.